<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>371</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 30px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid rgb(250, 50, 0)  ;
color: rgb(0, 0, 0);
background-color: rgb(235, 235, 235);
border-radius: 10px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailAddress_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom: 2px solid rgb(250, 50, 0)  ;
color: rgb(0, 0, 0);
background-color: rgb(235, 235, 235);
border-radius: 10px;</string>
     </property>
     <property name="placeholderText">
      <string>Enter Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: #00ABEE;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 94, 94);</string>
     </property>
     <property name="text">
      <string>Please fill out the following fields to login:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>UDSM ARIS 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="showPassword_checkBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>380</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#000</string>
     </property>
     <property name="text">
      <string>Show Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../ARIS-APP/img.qrc">:/img/png/udsm.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>password_lineEdit</zorder>
    <zorder>emailAddress_lineEdit</zorder>
    <zorder>login_pushButton</zorder>
    <zorder>showPassword_checkBox</zorder>
    <zorder>label</zorder>
    <zorder>label_11</zorder>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>750</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../ARIS-APP/img.qrc">:/img/bg/bg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>74</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#logout_pushButton{
	background-color: #00ABEE;
border-radius: 24px;
}

QPushButton#logout_pushButton:hover{
	background-color: #EE3248;
border-radius: 24px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../ARIS-APP/img.qrc">
      <normaloff>:/img/icon/close.png</normaloff>:/img/icon/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <zorder>bg</zorder>
   <zorder>frame</zorder>
   <zorder>logout_pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>emailAddress_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>showPassword_checkBox</tabstop>
  <tabstop>login_pushButton</tabstop>
  <tabstop>logout_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
